plugins/modules/swim_intent.py

```diff
@@ -28,7 +28,13 @@
   - cisco.dnac.intent_params
 author: Madhan Sankaranarayanan (@madhansansel)
         Rishita Chowdhary (@rishitachowdhary)
+        Abhishek Maheshwari (@abmahesh)
 options:
+  state:
+    description: The state of DNAC after module completion.
+    type: str
+    choices: [ merged ]
```

Comment:
Choices are merged and deleted

Reply:
For swim there is nothing like deleted that's what we discussed. It's all about updating even one API that is for Golden Image untagging is not exactly the delete operation.